.gitignore

```diff
@@ -68,6 +68,7 @@ examples/purchaseTester/.vscode/
 #
 examples/purchaseTester/ios/Pods
 examples/purchaseTester/ios/test_output
+**.Podfile.lock
```

Comment:
Hmm should this be `**/Podfile.lock`?

Reply:
🤦 

Also, not sure how I missed this comment, so double facepalm 🤦

Reply:
updated!